electrumsv/nodeapi.py

```diff
@@ -744,6 +744,13 @@ async def jsonrpc_getbalance_async(request: web.Request, request_id: RequestIdTy
     if len(parameter_values) > 1 and parameter_values[1] is not None:
         # INCOMPATIBILITY: It is not necessary to do a `node_RPCTypeCheckArgument` as the node does.
         minconf = get_integer_parameter(request_id, parameter_values[1])
+        # INCOMPATIBILITY: The node doesn't actually check for negative values but in our case,
```

Comment:
This comment is not really helpful, you are not saying why the negative values are meaningful so anyone reading this who isn't you right now while you remember it will have no idea.

Reply:
I'm not clear on what you want changed here. I thought it would be self-explanatory that one doesn't want the user inputting a negative number of confirmations - it doesn't really make any logical sense. Do you want me to just delete the comment and leave the `raise web.HTTPInternalServerError` for this?

Reply:
You are saying that there is a problem but you are not saying what the problem is. Don't you think that if you do this, you should be able to say what the problem is and why this is necessary? Not just "we don't like this so we're not going to take it but I won't say why we don't like this?".

Reply:
I have removed this check for a negative `minconf` now as the node doesn't check for it or return an error message and the result will be functionally equivalent to `minconf==0`